Makefile

```diff
@@ -211,6 +212,14 @@ db: # @HELP Opens the MySQL shell connected to the enduro development database.
 db:
 	mysql -h127.0.0.1 -P3306 -uroot -proot123 enduro
 
+pre-commit: # @HELP Check that code is ready to commit.
+	$(MAKE) lint
+	$(MAKE) golines
+	$(MAKE) gosec
+	$(MAKE) shfmt
+	$(MAKE) test-race
+	$(MAKE) workflowcheck
```

Comment:
You can declare deps with:

    pre-commit: lint golines gosec shfmt test-race workflow-check

It can also perform parallel execution, e.g. `make -j 2`.

Reply:
@sevein I am probably misunderstanding what you mean here, but when I do: 

```
pre-commit: # @HELP Check that code is ready to commit.
pre-commit: lint golines gosec shfmt test-race workflowcheck
	$(MAKE) lint \
	golines \
	gosec \
	shfmt \
	test-race \
	workflowcheck \
```

Each sub-target (e.g. lint, golines) runs *twice* and `pre-commit` takes twice as long. :(

If I change the rule to:
```
pre-commit: # @HELP Check that code is ready to commit.
pre-commit:
	$(MAKE) -j lint \
	golines \
	gosec \
	shfmt \
	test-race \
	workflowcheck \
```

Then `make pre-commit` runs a bit *slower* on average (~85s vs. ~82s) than when I omit the `-j` option. :confused: 

If I run `time make -j pre-commit` instead, then it runs about 10 seconds faster (~73s).  It doesn't seem to make any difference if I also include the `-j` parameter in the `pre-commit` command.